pkg/distributor/ha_tracker.go

```diff
@@ -157,7 +157,7 @@ func newClusterTracker(cfg HATrackerConfig) (*haTracker, error) {
 	}
 
 	if cfg.EnableHATracker {
-		client, err := kv.NewClient(cfg.KVStore, GetReplicaDescCodec())
+		client, err := kv.NewClient(RingNameForClient, cfg.KVStore, GetReplicaDescCodec(), reg)
```

Comment:
I think using the same name here would clash with the distributor ring name. We could just pass `distributor-hatracker` as ring client name. What do you think? /cc @pstibrany

Reply:
Makes sense